tutorials/spanruler_restaurant_reviews/configs/spanruler.cfg

```diff
@@ -0,0 +1,154 @@
+[paths]
+train = null
+dev = null
+vectors = "en_core_web_lg"
+init_tok2vec = null
+
+[system]
+gpu_allocator = null
+seed = 0
+
+[nlp]
+lang = "en"
+pipeline = ["tok2vec","ner"]
+batch_size = 1000
+disabled = []
+before_creation = null
+after_creation = null
+after_pipeline_creation = null
+tokenizer = {"@tokenizers":"spacy.Tokenizer.v1"}
+
+[components]
+
+[components.span_ruler]
```

Comment:
@kadarakos I did what you suggested. I haven't tested this yet, but you might want to take a look if the set up is correct.

Reply:
I decided to create two separate configs because I want to compare a NER only pipeline and one with SpanRuler. Is this the best way to do it (i.e., two separate configs?). cc: @kadarakos